### PR TITLE
test(storage): increase timeout for test_random_upgrade

### PR DESCRIPTION
**Pull request by mmilata**
Continuation of #2207.

### CODE REVIEW

**Comment by mmilata** [comment]
Brand new error: https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/2293416551
```
00:10:38 E           hypothesis.errors.DeadlineExceeded: Test took 2516.78ms, which exceeds the deadline of 2000.00ms
```

**Comment by andrewkozlik** [comment]
> cc @andrewkozlik, maybe we can loosen the deadline as well?

Agreed. It's basically the same thing, slow CI runners.

**Comment by matejcik** [comment]
fwiw @mmilata https://github.com/trezor/trezor-firmware/commit/89bf7bca7d18663fd6d60dec48f0f97fea8ef873 this seems nicer than explicitly adding timeouts where none were before

**Comment by mmilata** [comment]
Now gitlab isn't even showing the job at all, wut? Let me try again ...
